rust/noosphere-api/Cargo.toml

```diff
@@ -33,11 +33,11 @@ reqwest = { version = "0.11.15", default-features = false, features = ["json", "
 tokio-stream = "~0.1"
 tokio-util = "0.7.7"
 
-ucan = { version = "0.1.0" }
-ucan-key-support = { version = "0.1.0" }
+ucan = { workspace = true }
+ucan-key-support = { workspace = true }
 
-libipld-core = "~0.15"
-libipld-cbor = "~0.15"
+libipld-core = { workspace = true }
+libipld-cbor = { workspace = true }
```

Comment:
Oooooooh.
TIL!

Reply:
We've been incrementally migrating to workspace dependencies as we make updates / additions to existing manifests.